test/acceptance/behave/requirements.txt

```diff
@@ -6,4 +6,4 @@ requests==2.7.0
 xmltodict==0.9.2
 fabric==1.10.2
 pymongo==3.0.3
-git+https://visitor:visitor@pdihub.hi.inet/fiware/fiware-iotqaUtils@develop
\ No newline at end of file
+git+https://github.com/telefonicaid/iot-qa-tools.git
```

Comment:
if needed we can tag the code in a specific version for use it in your requirements, it is almost free to do it and use it like this: 

$ pip install git+https://github.com:telefonicaid/iot-qa-tools@picked_tag